spring-kafka-docs/src/main/antora/modules/ROOT/pages/testing.adoc

```diff
@@ -56,8 +56,11 @@ If this is not possible for some reason, note that the `consumeFromEmbeddedTopic
 Since it does not have access to the consumer properties, you must use the overloaded method that takes a `seekToEnd` boolean parameter to seek to the end instead of the beginning.
 ====
 
-NOTE: The `EmbeddedKafkaRule` JUnit 4 rule has been removed in version 4.0.
-For JUnit 4, you should use the `EmbeddedKafkaKraftBroker` directly or migrate to JUnit 5 with the `@EmbeddedKafka` annotation.
+[NOTE]
+====
+`Spring Kafka` no longer supports `JUnit4`.
+Migration to `JUnit Jupiter` is recommended.
```

Comment:
Those words don’t justify to be a code.
Plus, why stick `4` to `JUnit` word?

Also, the official project name is `Spring for Apache Kafka` and since we don’t use code wrapping here why do that for JUnit projects?